cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/realtime/sinks/RealtimeCubeSink.java

```diff
@@ -91,7 +116,6 @@ public int write(Iterable<StructuredRecord> objects, DataWriter dataWriter) thro
   public void initialize(RealtimeContext context) throws Exception {
     super.initialize(context);
     Map<String, String> runtimeArguments = context.getPluginProperties().getProperties();
-    datasetName = runtimeArguments.get(NAME_PROPERTY);
     transform = new StructuredRecordToCubeFact(runtimeArguments);
```

Comment:
Curious, do you need to pass all the plugin properties to StructurdRecordToCubeFact?